shared/umbrella/src/iosMain/kotlin/kmp/shared/umbrella/KotlinDateTime.kt

```
@@ -9,14 +8,19 @@ import kotlinx.datetime.toKotlinInstant
 import kotlinx.datetime.toLocalDateTime
 import kotlinx.datetime.toNSDateComponents
 import platform.Foundation.NSDate
+import kotlin.time.Instant
 
 // Force the companion object to be included in the framework. For example, upon adding the following code to Platform.kt in iosMain in a sample KMM project, the companion object was included in the headers:
 fun Instant.Companion.dummy(): Nothing = TODO()
```

detekt (1.23.8)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)